src/NServiceBus.AcceptanceTests/Recoverability/Retries/When_performing_slr_and_counting.cs

```diff
@@ -50,16 +50,14 @@ public RetryEndpoint()
             class ErrorNotificationSpy : IWantToRunWhenBusStartsAndStops
             {
                 Context testContext;
-                BusNotifications notifications;
+                Notifications notifications;
```

Comment:
Should we reformat this file to put the variables at the bottom or is that out of scope?